rust/xtask/src/main.rs

```diff
@@ -40,6 +43,7 @@ mod tasks {
         let filename = out_dir.join("agama.md");
         let mut file = File::create(&filename)?;
         file.write_all(markdown.as_bytes())?;
+        file.write_all(GENERATED.as_bytes())?;
```

Comment:
Shouldn't it be placed before the markdown content?

Reply:
I haven't checked how that would look like, before the title. Feel free to move it.

Reply:
It woud not be a valid md, so it is ok now.